giotto/mapper/visualization.py

```diff
@@ -277,12 +281,37 @@ def on_parameter_change(change):
 
             logger.info("Updating figure ...")
             with fig.batch_update():
-                node_trace, edge_trace, _, _, _ = \
+                node_trace, edge_trace, node_elements, node_colors, plot_options = \
                     _calculate_graph_data(
                         pipe, data, layout, layout_dim,
                         color_variable, node_color_statistic,  plotly_kwargs
```

Comment:
Two spaces before "plotly_kwargs"

Reply:
Fixed.

Reply:
Also fixed length-related linting problem due to the edited line above

Reply:
Thanks!